### PR TITLE
Get requirements without pinning

### PR DESCRIPTION
**Pull request by justcalamari**

### CODE REVIEW

File: conda_forge_tick/make_graph.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def get_attrs(name, i):
         host = list(req.get("host", []) if req.get("host", []) is not None else [])
         run = list(req.get("run", []) if req.get("run", []) is not None else [])
         req = build + host + run
-        req = set([x.split()[0] for x in req if x is not None])
+        req = set(re.split(" |=|>|<", x)[0].lower() for x in req if x is not None)
```
Comment (scopatz) [inline conda_forge_tick/make_graph.py]: I think it is better to pre-compile this regex.

**Comment by scopatz** [comment]
Thanks for putting this in @justcalamari!

**Comment by justcalamari** [comment]
Ready for review, @scopatz.

**Comment by scopatz** [comment]
Thanks @justcalamari!

**Pull request merged**